network-api/networkapi/wagtailpages/pagemodels/youtube.py

```diff
@@ -88,3 +88,20 @@ def get_context(self, request):
         return set_main_site_nav_information(self, context, 'Homepage')
 
     template = 'wagtailpages/pages/youtube_regrets_reporter_page.html'
+
+
+class YoutubeRegrets2021Page(FoundationMetadataPageMixin, Page):
+
+    template = 'wagtailpages/pages/youtube_regrets_reporter_page_2021.html'
```

Comment:
`youtube regrets reporter` is a browser extension, so let's make sure this is `youtube_regrets_page_2021.html`

Reply:
Ah, good catch. I wasn't aware of that. Template path and file name updated!